frontend-shared/styles/mixins/utils.scss

```diff
@@ -0,0 +1,11 @@
+/**
+ * Fast transition effect with fixed speed and ease.
+ *
+ * @param {string[]} $properties - CSS properties, such as background-color, to apply the
```

Comment:
This might be better documented as

`@param {...string} $properties`

per https://jsdoc.app/tags-param.html#parameters-with-properties

Though I just Googled that myself so it's not like I'm any kind of expert. But in any case: these strings are not passed in an Array...